website/data/docs-nav-data.json

```diff
@@ -458,10 +458,6 @@
             "title": "Ingress Controllers",
             "path": "k8s/connect/ingress-controllers"
           },
-          {
-            "title": "Ambassador Edge Stack Integration",
-            "href": "https://learn.hashicorp.com/tutorials/consul/load-balancing-ambassador"
-          },
```

Comment:
This should remain since this link is not broken.

Reply:
I see, so you still want the link to the learn tutorial. Gotcha, I thought this was simply missed during the cleanup.

Reply:
Yes there was a previous outdated Ambassador doc that was severely outdated. I just forgot to remove it from nav-data as well. Thanks!